### PR TITLE
new article and author

### PR DESCRIPTION
**Pull request by matstar2**

### CODE REVIEW

**Comment by rwp0** [comment]
Linking the previous PR for reference:
https://github.com/perladvent/perldotcom/pull/350

File: data/author/mathew-korica.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+   "name" : "Mathew Korica",
+   "bio" : "",
```
Comment (rwp0) [inline data/author/mathew-korica.json:3]: Could you please fill some `bio`?

**Comment by rwp0** [comment]
Thanks you very much for the article @matstar2 

I believe we would prefer cleaner commits without a `merge branch`:

image

Let me see if I can edit this PR if you wouldn't mind.

File: data/author/mathew-korica.json (hunk 1 above)
Comment (matstar2) [reply data/author/mathew-korica.json:3]: You can leave that blank (as is the case with other authors).

Comment (rwp0) [reply data/author/mathew-korica.json:3]: OK, we can do that

**Comment by matstar2** [comment]
I will try to provide a cleaner pull request.

**Pull request closed without merging**